docs/webhook.md

```diff
@@ -0,0 +1,57 @@
+# Webhook
+
+By default, Fleet utilizes polling (default: 15 seconds) to pull from a Git repo.However, this can be configured to utilize a webhook instead.Fleet currently supports Github,
+GitLab, Bitbucket, Bitbucket Server and Gogs.
+
+### 1. Configure the webhook service. Fleet uses `gitjob` service to handle webhook requests. Create an ingress that points to `gitjob` service.
```

Comment:
Fleet uses a `gitjob` service to handle webhook requests. Create an ingress that points to the `gitjob` service.